### ISSUE TITLE
USWDS - Bug: 508 opacity warning for ellipsis in pagination

### ISSUE THREAD

**Issue by ttnguyen2** [uswds/uswds#5188]
### Describe the bug

I got a opacity warning from ANDI for ... button in pagination. Can we have the color darker to pass ANDI test?

### Steps to reproduce the bug

1. Go to https://designsystem.digital.gov/components/pagination/
2. Have ANDI installed on browser to test 508
3. Open color Contrast drop down on ANDI to test color contrast on pagination page
4. See the opacity warning for the grey ... of pagination

### Expected Behavior

increase the opacity of ... to pass ANDI color contrast test

### Screenshots

![pagination](https://user-images.githubusercontent.com/101652844/226041112-abf88631-a662-44e8-a49f-3c8df0d03de4.PNG)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22) for duplicate bug reports.

**Comment by amyleadem** [comment]
Confirming that the current color contrast for ellipses is 3.31:1, which fails standards for small text. 
Recommendation: We should remove the opacity and convey "lightness" in a different way. 

Note: tested the color rgba(27,27,27,0.5)
